recipe/meta.yaml

```diff
@@ -233,6 +231,8 @@ outputs:
         - pytest-timeout
         - pytest-xdist    # [not (win and python_impl == "pypy")]
         - mpmath
+        # needed for scipy.datasets
+        - pooch
```

Comment:
Moving to `test` `requires` as requested in https://github.com/conda-forge/scipy-feedstock/issues/247#issuecomment-1710916764.